package.json

```diff
@@ -22,16 +29,21 @@
 	},
 	"homepage": "https://github.com/LeoPlatform/bus#readme",
 	"dependencies": {
+		"aws-sdk": "^2.466.0",
```

Comment:
This is added as an external dependency in lambda so it isn't need in dependencies.  It should probably be in devDependencies